### DISCUSSION TITLE
why  clickhouse local -qf a.sql  reports error

### DISCUSSION THREAD

**Discussion by l1t1** [ClickHouse/ClickHouse#49059] [Q&A] [answered]
but clickhouse local < a.sql works

**Comment by l1t1** [comment]
root@localhost:/data/data/com.termux/files/home# ./ck local < select42.txt                                
42                                                                                                        
root@localhost:/data/data/com.termux/files/home# ./ck local -qf 'select42.txt'                            
Code: 36. DB::Exception: Positional options are not supported. (BAD_ARGUMENTS)                            root@localhost:/data/data/com.termux/files/home# cat select42.txt                                         
create table a engine=Log as select 42 a;                                                                 
select * from a;

**Comment by l1t1** [comment]
--queries-file works
but 
https://github.com/ClickHouse/ClickHouse/pull/15930/commits/f5d1a9e59a1ce9054a9dc96a74bb9bee61fb9ccc
says
-   `-qf` `--queries-file` - file path with queries to execute. You must specify either `query` or `queries-file` option.

**Comment by alexey-milovidov** [comment, marked as answer]
Only `--queries-file` option is supported.
There is no `-qf` option. The documentation was updated.
